services/movie_session.py

```diff
@@ -42,3 +42,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling exceptions such as `MovieSession.DoesNotExist` when deleting a movie session by ID. This will prevent the application from crashing if an invalid ID is provided.